backends/lib/lib.go

```
@@ -0,0 +1,518 @@
+	for _, name := range defaultLibNames {
+		if h, err := purego.Dlopen(name, purego.RTLD_LAZY|purego.RTLD_LOCAL); err == nil {
+			_ = purego.Dlclose(h)
+			return name, nil
+		}
+	}
+	for _, path := range defaultLibPaths {
+		if info, err := os.Stat(path); err == nil && !info.IsDir() {
+			return path, nil
+		}
+	}
+	return "", errors.New(
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)